templates/modelExpt/run_modelComp_expt.m

```diff
@@ -0,0 +1,170 @@
+function expt = run_modelComp_expt(expt,bTestMode)
+% RUN_MODELCOMP_EXPT - A simple AAF compensation experiment where F1 is
+% shifted up on 20% of trials, F1 shifted down on 20%, and no shift 60%.
+%
+% Set bTestMode input argument to 1 to run a shortened version.
+%
+% Based around run_cerebAAF_expt and run_modelExpt_expt
+
+% v1 2023-10 - Chris Naber
+
+%% Default arguments
+
+if nargin < 1, expt = []; end
+if nargin < 2 || isempty(bTestMode), bTestMode = 0; end
+
+%% Set up general experiment parameters
+expt.name = 'modelComp';
+expt.trackingFileLoc = 'experiment_helpers'; % Where the OST/PCF files are kept (for audapter_viewer)
+expt.trackingFileName = 'measureFormants'; % What the files are called (does not include Working/Master)
+refreshWorkingCopy(expt.trackingFileLoc,expt.trackingFileName);
```

Comment:
Remove if this experiment doesn't use OSTs. Could make a model experiment that does.

Reply:
correct - removed